search/includes/classes/class-search.php

```diff
@@ -1017,7 +1021,7 @@ public function is_url_query_cacheable( string $url, $args ): bool {
 			return $is_cacheable;
 		}
 
-		$is_cacheable = (bool) preg_match( '#/_(search|mget|doc)#', $url );
+		$is_cacheable = (bool) preg_match( '#/_(search|mget|doc)#', $url ) && in_array( $args['method'], [ 'DELETE', 'PUT' ], true );
```

Comment:
Aren't search requests a `POST` request?

Reply:
Or was it intended for an `! in_array()` based on the `Skip DELETE and PUT from cacheable methods` comment?

Reply:
typo!